### ISSUE TITLE
Couldn't parse arguments. java.lang.NumberFormatException: For input string: "". [JVDBG] Error, jsId = null for jsEventHandler

### ISSUE THREAD

**Issue by samyc1201** [gluonhq/substrate#767]
I use JavaFX webview to load a set of local html/javascript file.

```
WebView webView = new WebView();

String indexFilePath = Paths.get(getPublicPath(), "demo", "index.html").toString();

webView.getEngine().load("file://" + indexFilePath);

VBox root = new VBox(webView);
root.setAlignment(Pos.CENTER);
Scene scene = new Scene(root, 640, 480);
//        scene.getStylesheets().add(HelloFX.class.getResource("styles.css").toExternalForm());
stage.setScene(scene);
stage.show();
```

The files can be loaded properly, and the javascript looks running fine. I use safari debugger to debug, no error throw in safari debugger console.

But in the logs trace (from maven logs), it repeat printing the error message below while I navigate through the page:

```
[Mon Aug 31 12:59:03 SGT 2020][INFO] [SUB] Couldn't parse arguments. java.lang.NumberFormatException: For input string: ""
[Mon Aug 31 12:59:03 SGT 2020][INFO] [SUB] [JVDBG] Error, jsId = null for jsEventHandler

```

Something is wrong with the javascript bridge? Looks like it is printing out from Substrate.

Below is the version I am using:

```
<maven.compiler.source>11</maven.compiler.source>
<maven.compiler.target>11</maven.compiler.target>
<javafx.version>14</javafx.version>
<attach.version>4.0.7</attach.version>
<client.plugin.version>0.1.31</client.plugin.version>
```

**Comment by jperedadnr** [comment]
This is a known issue, it is being addressed in #686

**Comment by jperedadnr** [comment]
This issue has been already fixed
